yarn-project/archiver/src/archiver/archiver_store.ts

```diff
@@ -38,6 +39,21 @@ export interface ArchiverDataStore {
    */
   addBlocks(blocks: L2Block[]): Promise<boolean>;
 
+  /**
+   * Append new block bodies to the store's list.
+   * @param blockBodies - The L2 block bodies to be added to the store.
+   * @returns True if the operation is successful.
+   */
+  addBlockBodies(blockBodies: Body[]): Promise<boolean>;
+
+  /**
+   * Gets block bodies that have the same txHashes as we supply.
+   *
+   * @param txsHashes - A list of txsHashes that correspond to the body hashes
```

Comment:
Agree. I don't think we should ever really be talking about body hashes, because that becomes a bit confusing whether it is the full body or what it is really talking about. 

I prefer `txsHash` over `bodyHash` because it is not the full body. But maybe it should be something different entirely since it can also quite easily be mistyped as `txHash` or people talking about it like that. 

I think we should alter it, but likely in a different pr such that we can also deal with all the calldata ones ☠️ I can take a look at it after reviewing here.